api/envoy/admin/v3/listeners.proto

```diff
@@ -34,4 +34,9 @@ message ListenerStatus {
   // The actual local address that the listener is listening on. If a listener was configured
   // to listen on port 0, then this address has the port that was allocated by the OS.
   config.core.v3.Address local_address = 2;
+
+  // The additional actual local addresses that the listener is listening on. If a listener was configured
+  // to listen on port 0, then this address has the port that was allocated by the OS.
+  // [#not-implemented-hide:]
```

Comment:
Can you update this to ref link against the listener fields that this comes from? Same above? It's a little confusing what this means without the other context.

Reply:
got it, it is same with above.